chatbot/db/db.js

```diff
@@ -41,14 +53,17 @@ module.exports.getUnrespondedSessionWithButtonId = async function(buttonId, clie
     const query = "SELECT * FROM sessions WHERE button_id = $1 AND state != $2 AND state != $3 AND state != $4"
     const values = [buttonId, STATES.WAITING_FOR_CATEGORY, STATES.WAITING_FOR_DETAILS, STATES.COMPLETED]
     const { rows } = await client.query(query, values)
-   
+
+    let session = await createSessionFromRow(client, rows[0])
+
     if(!transactionMode) {
```

Comment:
Does this need to be in a finally block? i.e. would the client be released if an error occured during this method?